lib/generators/curation_concerns/install_generator.rb

```diff
@@ -101,8 +101,11 @@ def resource_types_config
       copy_file "config/authorities/resource_types.yml", "config/authorities/resource_types.yml"
     end
 
-    def simple_form_bootstrap
-      generate 'simple_form:install --bootstrap'
```

Comment:
I can see it either way, though I have a slight preference for leaving it as is.

When simple_form changes underneath us, we should change. I'm loath to guess at how it will change. For instance, what if they decide to break simple_form.rb into two files with different names? Then we copy over two files that potentially aren't used anymore and may be overridden in part or in full by the new files.